src/App.js

```diff
@@ -1,18 +1,73 @@
 // TODO: translate everything to SLOVAK language
+// TODO: add dark mode to all pages
+// TODO: make dark mode switching more smooth
+// TODO: generate MIT license here: https://license-generator.intm.org/
 
-import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import { useDefaultValue } from './hooks/useDefaultValue';
+import { load as loadCookie, save as saveCookie } from 'react-cookies';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import { Home } from './components/Home';
 import { Login } from './components/Login';
+import { _404 } from './components/404';
+import { About } from './components/About';
+import './styles/Global.css';
+
+// got a better idea? write it down here
+let useTheme = className => className;
```

Comment:
Is there any reason why this is devided? If not, please make it into only 1 statement
```suggestion
const useTheme = (className) => { return `${className} ${darkMode ? `${className}-dark` : ""}` };
```

Reply:
Yes, there is. Basicly `darkMode` is only in the App component scope, not in whole file. If it was defined like that OUTSIDE App it would throw an NameError (variable/function with that name doesnt exist).

Reply:
Ok